gui/default/assets/css/overrides.css

```diff
@@ -351,11 +348,12 @@ ul.three-columns li, ul.two-columns li {
         border-radius: 2px;
     }
 
-    *[language-select] {
+    li[language-select] {
         position: static !important;
     }
 
-    *[language-select] > .dropdown-menu {
+    li[language-select] > .dropdown-menu {
+        column-count: auto;
```

Comment:
Is it ever 2 as above? Should the column-gap then also be 0 for consistency?

Reply:
The `column-gap` is used without any media queries, so it applies here too, but `column-count: auto` in this case actually just disables columns altogether (with `auto` being the default value). In other words, we've got 1 column in `@media (max-width: 768px)` and 2 columns otherwise.

I'd prefer to do it in the opposite direction, i.e. 1 column by default and then use a `min-width` based media query to switch to 2 columns on wider screens only, but doing so would require quite a lot of code changes. I don't want to do a revolution in this small commit, so I think the current approach is a decent compromise (and with proper browser support too).